### PR TITLE
Adding History Page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Context/History.context.js
- [ ] src/Pages/HistoryPage/HistoryPage.jsx
- [ ] src/Reducer/Reducer.js

## What Not To check
- [ ] Do Not Check Any Other Files

live preview,
https://stormplay.netlify.app/

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Looking good. You can go ahead with merging !!

File: src/Pages/HistoryPage/HistoryPage.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Loader, VideoCard } from "Components";
```
Comment (mikesch803) [inline src/Pages/HistoryPage/HistoryPage.jsx:1]: Nothing in history, page is also blank , please check

**Review by mikesch803 — CHANGES_REQUESTED** [review]
your code is clean, everything looks good, just check on UI, its not updating

File: src/Pages/HistoryPage/HistoryPage.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/HistoryPage/HistoryPage.jsx:1]: can you check again for me both preview link and the jsx page is working

Comment (b0llu) [reply src/Pages/HistoryPage/HistoryPage.jsx:1]: can you check again both live preview and jsx file is showing for me

Comment (mikesch803) [reply src/Pages/HistoryPage/HistoryPage.jsx:1]: i checked, website is not working from my end

**Pull request merged**